Integrations/FidelisElevateNetwork/README.md

```diff
@@ -0,0 +1,1283 @@
+Automate Detection and Response to Network Threats and data leakage in your organization with Fidelis Elevate Network Integration.
+This integration was integrated and tested with version xx of Fidelis Elevate Network
```

Comment:
Please replace xx with the version we used for testing

Reply:
done